src/sandwich-menu/components/sandwich-menu.jsx

```diff
@@ -0,0 +1,69 @@
+import MUIDataTable from 'mui-datatables'
+import React, {useEffect, useState} from "react";
+import {Box, Grid, Toolbar, Badge, IconButton} from "@material-ui/core";
+import AddShoppingCartIcon from '@material-ui/icons/AddShoppingCart';
+import {useOnAddCart} from "../containers/hooks/use-on-add-cart";
+import {Title} from "../containers/details/title"
+import {ShowDetails} from "../containers/details/body";
+import {columns} from "../containers/table-columns";
+import {useDispatch, useSelector} from "react-redux";
+import {getAllSandwiches} from "../actions";
+
+export const Menu = () => {
+
+    const [selected, setSelected] = useState([]);
+    const dispatch = useDispatch()
+    useEffect(() => dispatch(getAllSandwiches()),[])
+    const data = useSelector(selector)
+    const onClick = useOnAddCart(selected,setSelected, selector);
+    
+    const options = {
```

Comment:
We can maybe extract this huge `options` object in a different hook.

Reply:
Done